src/index.html

```diff
@@ -14,6 +14,8 @@
     <link rel="stylesheet" type="text/css" href="https://cartodb-libs.global.ssl.fastly.net/cartodb.js/v3/3.15/themes/css/cartodb.css" />
     <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/selectize.js/0.12.4/css/selectize.default.min.css" />
     <link rel="stylesheet" type="text/css" href="styles/main.min.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.10/css/all.css" integrity="sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg"
```

Comment:
Can we just thump in the SVG directly like DC does? Would prefer to avoid adding a new dependency if we're just using one icon.

Reply:
Let's remove this; don't think we're using it anymore.